### ISSUE TITLE
Add option to report worker for user

### ISSUE THREAD

**Issue by 0kSure** [HumanbiOS/telegram_bot_frontend#49]

**Comment by Poolitzer** [comment]
Where is this report supposed to go, worker group?

I pictured this: A /report command. If send by the user, check if they are in a conversation. If yes, tell them this is serious business. Then ask them to reply to the reported message and write their report or send a screenshot of the message history to be reported and add their report as caption to the picture. Does this sound good?

**Comment by 0kSure** [comment]
Like the idea! Reports should also be stored serverside so things can be banned.

**Comment by Poolitzer** [comment]
Fair enough. I can store it on the server, but should the bot also notify the workers about this report? Or at least notify someone?
